### ISSUE TITLE
Implementation checker: Show only "fail"?

### ISSUE THREAD

**Issue by pjotrek-b** [MediaArea/MediaConch#212]
When analyzing a Matroska file that fails the implementation check, is there a way to show only the parts that "fail"?

I currently download the XML/HTML and search, but thought I'd ask... :smile:

**Comment by ablwr** [comment]
Yeah you can set `--ImplementationVerbosity=V` (or  `-iv V`) and the default is 5, the most verbose, but 4 will show only fails and n/a!

**Comment by JeromeMartinez** [comment]
If you ask, it may mean that the UI is not good enough :(.
Because I understand you want to have a verbosity option, and there is already a verbosity dropbox that fits your need, in the implementation report window.
But maybe a verbosity number is not good, we should display e.g. "errors only".

Or did I miss something?

**Comment by ablwr** [comment]
I mean, this is for the whole report though.

**Comment by ablwr** [comment]
@JeromeMartinez I would agree that the verbosity feature is not clear

**Comment by pjotrek-b** [comment]
If you could add "errors only" string next to the number (like it has for 0 and 5), I think it'd be clearer :smile:

btw: I just noticed that there is no "verbosity-like filter" for policies, so one cannot see only "failed" there.

(Using MediaConch Online version, because most recent .deb (17.12) on my machine (Xubuntu 16.04.3) freezes at 100% CPU, regardless of file. Will try to reinstall from MediaArea repositories...)

**Comment by JeromeMartinez** [comment]
We'll add some text.
I changed the issue title for policy checker verbosity.

For the Ubuntu issue we'll do some test soon with next release, for being sure all is OK on our side.

**Comment by pjotrek-b** [comment]
Great! Thanks :)
